RDD with Discrete Running Variables.md

```diff
@@ -0,0 +1,134 @@
+
+p-value = $\frac{\hat{\tau} - B}{\frac{\sigma}{\sqrt{N}}}$ 
+
+The resulting BSD confidence interval has the following properties: 
+
+- It is honest with respect to the function space $\Mu(K)$. 
+
+- It is valid regardless of whether the running variable is discrete or continuous. 
+
+- It is valid for any choice of bandwidth.  
+
+- It allows the researcher to choose a bandwidth minimising the size of the interval in order to obtain the tightest possible confidence interval without invalidating the results.  
+
+The main difficulty with implementing BSD confidence intervals is the necessity to choose, ex ante, the bound $K$ on the absolute value of the second derivative of the CEF $\mu$ in $\Mu$. In order to do this, it is essential to use prior, context-specific knowledge regarding the relationship between the running and outcome variables. Intuitively, choosing a $K$ close to 0 is equivalent to assuming that the CEF is close to being linear, while selecting a higher $K$ implies assuming a higher-polynomial CEF.  
+
+Although it is not possible to obtain a higher bound for $K$ from the data used by the researcher for the treatment effects estimation, it is possible to derive a lower bound for $K$ based on the data using the method described in Armstrong and Kolesár (2018). We can then at least be aware of which values of $K$ are too small to be compatible with our data.  
+
+## Implementing Honest BSD confidence intervals in R
+
+In this section, I demonstrate a simple implementation of BSD confidence intervals in R using the [RDHonest](https://github.com/kolesarm/RDHonest/blob/master/doc/RDHonest.pdf) package. The package includes some datasets, such as _cghs_, which relates to Oreopoulos's (2006) study of the effects of a change in the minimum school leaving age in Britain on earnings. Oreopoulos uses age as the discrete running variable of his study. 
+
+{{% codeblock %}}
+```R
+#install the RDHonest package
+install.packages("remotes")
+remotes::install_github("kolesarm/RDHonest")
+#load the RDHonest package into memory
+library(RDHonest)
+#view the data
+head(cghs)
+```
+{{% /codeblock %}}
+
+We can first visualise the regression discontinuity by using the RDScatter function built into the package. 
+
+{{% codeblock %}}
+```R
+#plot the regression discontinuity using the RDScatter function
+#the option propdotsize allows us to make dot size proportional to the number of observations for each value of the running variable
+RDScatter(log(earnings) ~ yearat14, data = cghs, cutoff = 1947, avg = Inf, xlab = "Year aged 14", ylab = "Log earnings", propdotsize = TRUE)

```

Comment:
include screenshots of the output to give the reader an idea of how it would look like